pkg/cmd/codespace/common.go

```diff
@@ -57,20 +56,8 @@ func (a *App) StopProgressIndicator() {
 	a.io.StopProgressIndicator()
 }
 
-type liveshareSession interface {
```

Comment:
This interface doesn't seem to really be that useful? Maybe there's a reason for having it, but using `*liveshare.Session` seems works the same.

Reply:
Agreed that this interface wasn't useful. Maybe in the past it was used to pass around a fake implementation of liveshare session, but it's a better approach that individual commands that depend on a session define their own interface for the session methods that they expect.